### DISCUSSION TITLE
Middleware are not working fine

### DISCUSSION THREAD

**Discussion by alejo9424** [adonisjs/core#3179] [Help]
Hello i have a little problem with middleware.

this is my routes.js configuration i want the group has the middleware and this working fine.
![image](https://user-images.githubusercontent.com/48483048/133502644-1d7cfc2d-68d7-4eb6-accd-53c7343f9ba0.png)

 The problem is that the middleware is taking the others routes.
![image](https://user-images.githubusercontent.com/48483048/133503116-87b6e23b-1d33-4ebb-8c2b-978e1ef6d151.png)

any suggestions for me?

**Comment by RomainLanz** [comment]
Hey @alejo9424! 👋 

When you are using the `node ace list:routes` command, do you see your middleware used on those routes?
